protocols/v2/subprotocols/common-messages/Cargo.toml

```diff
@@ -14,7 +14,8 @@ binary_sv2 = {version = "0.1.*", path = "../../../../protocols/v2/binary-sv2/bin
 const_sv2 = {version = "0.1.*", path = "../../../../protocols/v2/const-sv2"}
 quickcheck = { version = "1.0.3", optional=true }
 quickcheck_macros = { version = "1", optional=true }
+serde_repr = {version= "0.1.10", optional=true}
```

Comment:
Can you explain why we need this?

Reply:
I had problem in parsing type Protocol, so I used serve_repr
https://github.com/stratum-mining/stratum/blob/main/protocols/v2/subprotocols/common-messages/src/setup_connection.rs#L287

Reply:
https://github.com/stratum-mining/stratum/pull/383/commits/db1936bc365dd6be0ab13fc13e52de31763eea2c